onnxruntime/contrib_ops/rocm/bert/skip_layer_norm.cc

```diff
@@ -96,6 +100,14 @@ Status SkipLayerNorm<T>::ComputeInternal(OpKernelContext* ctx) const {
   int64_t element_count = input_dims[0] * sequence_length * hidden_size;
   typedef typename ToHipType<T>::MappedType HipT;
 
+  if (tuning_) {
+    static SkipLayerNormTunableOp<T> op;
+    op.EnableTuning();
+
+    SkipLayerNormParams<T> op_params(stream, output, input, skip, gamma, beta, bias, epsilon, ld, element_count);
```

Comment:
Is this code correct? I think the original code with this inside Kernel Launch function is correct, as we need to pass in input->Data<T>() instead of input.

Reply:
Put this inside KernelLaunch function or this place neither is the best way. Instead of adding more files to support, I change it to original code. May be refactor in other PR.